reference/managed-delivery/resource-status/index.md

```diff
@@ -0,0 +1,160 @@
+---
+layout: single
+title:  "Resource Status"
+sidebar:
+  nav: reference
+---
+
+{% include toc %}
+
+*If you're not sure what a Managed Resource is, check out our [overview of Managed Delivery](/reference/managed-delivery) first.*
```

Comment:
(Cosmetic) How do you feel about using :bulb: for tips like these?

Reply:
You know I have trouble saying no to emojis

Reply:
Big fan of having consistent icons to help break up the monotony of text.